### PR TITLE
Player v3 studio

### PR DESCRIPTION
**Pull request by yairans**
new studio app for kaltura player v3
FEC-7395

### CODE REVIEW

File: app/js/app.js [hunk 1]
```diff
@@ -251,10 +251,10 @@ KMCModule.config(['$routeProvider', '$locationProvider', '$httpProvider', '$tool
         }
     });
 	var kmc = window.parent.kmc;
-    if (kmc && kmc.vars.studio.showFlashStudio === false){
+    if (kmc && kmc.vars.studioV3.showFlashStudio === false){
```
Comment (amirch1) [inline app/js/app.js]: @yairans - what about a situation where the user have permissions to v2 and v3 studios but not v1? The subNav will be hidden and he will not be able to navigate between v2 and v3 studios.

File: app/js/app.js [hunk 2]
```diff
@@ -251,10 +251,10 @@ KMCModule.config(['$routeProvider', '$locationProvider', '$httpProvider', '$tool
...
         $(".kmcSubNav").hide();
     }
-    if (kmc && kmc.vars.studio.showHTMLStudio === false){
+    if (kmc && kmc.vars.studioV3.showHTMLStudio === false){
```
Comment (amirch1) [inline app/js/app.js]: @yairans  - where is the handing of the v3 studio permissions?

File: app/js/controllers/EditCtrl.js [hunk 3]
```diff
@@ -658,23 +677,33 @@ KMCMenu.controller('EditCtrl', ['$scope','$http', '$timeout','PlayerData','Playe
...
 			window.parent.studioDataChanged = false; // used when navigating away from studio
 			if ($scope.playerData.config.plugins.playlistAPI && $scope.playerData.config.plugins.playlistAPI.plugin){
 				$scope.addTags(['html5studio','playlist']); // set playlist tag
 			}else{
-				$scope.addTags(['html5studio','player']); // set player tag
+				$scope.addTags(['kalturaPlayerJs','player']); // set player tag
+			}
+			if (!$scope.playerData.autoUpdate && PlayerService.getPlayerVersion($scope.playerData) === '{latest}') {
+				//should load the latest KalturaPlayer to figure out the version number
+				PlayerService.loadKalturaPlayerScript($scope.playerData, function () {
```
Comment (amirch1) [inline app/js/controllers/EditCtrl.js]: @yairans - why do you need the player version? Isn't {latest} can be used?

File: app/js/services/services.js [hunk 4]
```diff
@@ -710,29 +723,9 @@ KMCServices.directive('loadingWidget', ['requestNotificationChannel', function (
...
-			if (typeof data == 'object') // json is OK
-				deferred.resolve(result.data);
-			else {
-				cl('JSON parse error of playerFeatures');
-				deferred.reject(false);
-			}
-		}, function (reason) {
-			deferred.reject(reason);
-		});
+      return $http.get('js/services/v3Properties.json').then(function(result){
```
Comment (amirch1) [inline app/js/services/services.js]: @yairans - is this going to stay hard-coded? If so, no need to wait for the API promise...

File: app/js/controllers/EditCtrl.js (hunk 3 above)
Comment (yairans) [reply app/js/controllers/EditCtrl.js]: in v3 studio you can freeze the version to not be updated automatically. for this we have to extract the version.

**Pull request merged**